project01/paulo/direction.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cinder/gl/gl.h>
+
+enum class Direction
+{
+  None,
```

Comment:
I am not sure about this `None` direction. How is it possible, in a physical world? You can look at any direction, you will always be looking somewhere, wont you? On a more practical note, this is slightly annoying in code as you need to handle this value in code, in every switch statement, function, etc. It adds complexity in my opinion. What is the upside?